docs/topics/spiders.rst

```diff
@@ -371,6 +371,28 @@ CrawlSpider
       :class:`~scrapy.item.Item` object, a :class:`~scrapy.http.Request`
       object, or an iterable containing any of them.
 
+   .. method:: process_results(response, results)
+
+      This overridable method is called for each result (item or request)
+      returned by the spider, and it's intended to perform any last time
+      processing required before returning the results to the framework core,
+      for example setting the item GUIDs. It receives a list of results and
+      the response which originated that results. It must return a list
+      of results (Items or Requests).
+
+   .. method:: _requests_to_follow(response)
+
+      This method iterates over each of the spider's rules, extracts the links
+      matching each case, filters them (if needed), and returns a list of unique
+      requests per response.
+
+   .. method:: _response_downloaded(response)
```

Comment:
It seems there is still duplication. Do you think it is possible to make http://www.sphinx-doc.org/en/stable/ext/autodoc.html#directive-automethod work for this?